### DISCUSSION TITLE
Unable to increase memory limit on Serverless Function

### DISCUSSION THREAD

**Discussion by Ryanjso** [vercel/vercel#10502] [Help]
### Question

Hello, can anyone help me figure out why my Serverless Functions aren't deploying with more memory. I have tried setting it in both `vercel.json` and `export config` in the api itself. Neither seem to change what is shown on the function page. It's possible it does have more memory but the page itself is incorrectly showing 1024, I'm not quite sure how I would check that. For context I am on the Vercel Pro Trial so should have access to more memory.

<img width="1247" alt="Screenshot 2023-09-13 at 12 10 49 PM" src="https://github.com/vercel/vercel/assets/39172778/31f9113a-6764-4ce8-9d13-0258c1c93d9b">
<img width="387" alt="Screenshot 2023-09-13 at 12 11 03 PM" src="https://github.com/vercel/vercel/assets/39172778/151b9329-6a45-4fbe-bc44-a3cf453989ec">
<img width="332" alt="Screenshot 2023-09-13 at 12 11 13 PM" src="https://github.com/vercel/vercel/assets/39172778/f6f8c5d9-d97d-4834-952e-e40145edc17d">

**Comment by MathiasGruber** [comment]
I'm experiencing the same thing

**Reply by Ryanjso** [reply]
Hey I eventually figured out the issue, I have my app inside an src folder so I needed to add that to the key like so:
```"functions": {
    "src/pages/api/instance/sync.ts": {
      "maxDuration": 120
    }
 ```

The issue is if you write a function path that doesn't exist like `"pages/api/doesntexist.ts"` Vercel will throw an error that the function doesn't exist, but in my case the function did exist, my path just didn't go correctly to it so Vercel didn't throw any errors. It seems Vercel's code that checks if the path exists has different rules than their code that actually handles the configuration changes, so it was silently failing.

Hope this helps :)

**Reply by MathiasGruber** [reply]
Thank god, this solved it, this was so annoying. Thanks for responding, truly appreciated! 👍
